spec/tanker/identity_spec.rb

```diff
@@ -9,7 +9,9 @@
     @app = test_app
     @user_id = 'b_eich'
     @user_email = 'brendan.eich@tanker.io'
+    @user_phone = '+33178919191'
```

Comment:
Lol, it's not a mobile phone.

Reply:
C'est la ligne fixe de l'Archevêché de Paris! Remplacé par un 06 placeholder